XYZDrivers/src/main/java/com/xyzdrivers/repositories/MembersRepo.java

```diff
@@ -25,16 +32,22 @@
             results = this.sqlService.retrieve("MEMBERS");
             //parse members data
             for (Object[] memberData : results) {
+                String dateTime = df.format(memberData[3]);
+                String dateTime2 = df.format(memberData[4]);
+
+                dob.setTime(df.parse(dateTime));
+                dor.setTime(df.parse(dateTime2));
+
                 Member member = new Member(memberData[0].toString(), //id
                         memberData[1].toString(), //name
                         memberData[2].toString(), //address
-                        LocalDate.parse(memberData[3].toString()), //dob
-                        LocalDate.parse(memberData[4].toString()), //dor
+                                             dob, //dob
```

Comment:
There's a sneaky bug here just waiting to show itself during demo. You're inserting the same two calendar instances into every member object you create and they're not passed by value, they're passed by reference. Every member will appear to have the same DOB/DOR as the last member retrieved.

Move your Calendar declarations into the loop and call Calendar.getInstance() every time so you don't re-use instances